### DISCUSSION TITLE
asuka theme 3.5

### DISCUSSION THREAD

**Discussion by gigi798** [mathoudebine/turing-smart-screen-python#555] [Themes]
i have ported this theme found on turzx site 

to set gpu and cpu name edit theme.yaml

![preview](https://github.com/user-attachments/assets/c93eb019-aabb-4f14-9dce-3fb5fc9237fa)

[eva.zip](https://github.com/user-attachments/files/16630641/eva.zip)
ryzen font 
[ryzen.zip](https://github.com/user-attachments/files/16642965/ryzen.zip)

**Comment by madkoding** [comment]
![Screenshot](https://github.com/user-attachments/assets/492dceee-e3ac-4b9b-ab87-1a9fc25a12e6)

My humble config
Great theme

**Comment by Popsicle7791** [comment]
Works great! I actually ran it with the same modification as @madkoding

**Reply by gigi798** [reply]
which  font is that ?

**Reply by Popsicle7791** [reply]
Oh I didn't even notice the font changed. I used your font. I changed the progress bar and stats colour to match the eva panel.

**Comment by sysyphus** [comment]
Nice one~!

**Comment by heavyoak** [comment]
I like it but I wish there was a Rei version.

**Reply by madkoding** [reply]
I wish too but, too busy to make it xD

Soon (TM)

**Comment by ScareVVare** [comment]
also wished for a Rei-version, so i did some quick'n'dirty copycat version .. enjoy the questionable quality.

*Update.v002 - fragment cleanup, USAGE-tag moved higher, character in foreground now, eva-tribal color correction

![preview](https://github.com/user-attachments/assets/bda1d752-37b9-4a3f-ae15-524be6751662)
[eva-blue.v002.zip](https://github.com/user-attachments/files/19713553/eva-blue.v002.zip)

**Reply by heavyoak** [reply]
thats actually quite good.
